worlds/alttp/__init__.py

```diff
@@ -724,6 +724,23 @@ def get_pre_fill_items(self):
                         res.append(item)
         return res
 
+    def fill_slot_data(self):
+        slot_data = {}
+        if not self.multiworld.is_race:
+            for option_name in alttp_options:
```

Comment:
I don't like that this is pulling every option in the options system. I know that isn't currently a lot for LTTP, but there's still some there that I doubt you need, like palette shuffle, and this will only get worse as the options get moved over.

Reply:
> I don't like that this is pulling every option in the options system. I know that isn't currently a lot for LTTP, but there's still some there that I doubt you need, like palette shuffle, and this will only get worse as the options get moved over.

which version would you  prefere then? 
having a static list, iterating over it and adding it to slot_data?
OR simply having ~20 static, basically identical, lines for each option i need/want for the tracker?

Reply:
> having a static list, iterating over it and adding it to slot_data?

This exactly. you make it sound like a bad thing? fwiw #993 makes this significantly easier, but we don't have that yet.

Reply:
Oh. no. I have nothing against that solution and never had. 

I will change that when i'm back home.

Reply:
is that what you intended it look like?